From 712ef3c8c1632a538526ece64be05a3763516b34 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Mon, 16 Dec 2024 22:30:45 -0700
Subject: [PATCH] fix: Update the Makefile to avoid hardcoded f77

* Apply the Makefile patches from QCDLoop v1.981's vendored version of FF.
   - c.f. https://qcdloop.fnal.gov/
---
 src/ff/Makefile | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/ff/Makefile b/src/ff/Makefile
index f47b45e..6869326 100644
--- a/src/ff/Makefile
+++ b/src/ff/Makefile
@@ -1,4 +1,5 @@
-FFLAGS        = 
+FC            = gfortran
+FFLAGS        = -std=legacy
 
 LFLAGS	      = $(FFLAGS)
 
@@ -7,7 +8,7 @@ DEST	      = /usr/local/lib
 HDRS	      = aa.h \
 		ff.h
 
-LIBRARY	      = ff.a
+LIBRARY	      = libff.a
 
 MAKEFILE      = Makefile
 
@@ -162,7 +163,7 @@ test:		npointes
 		diff -w npointes.out npointes.sun
 
 npointes:	npointes.o $(LIBRARY)
-		f77 $(LFLAGS) -o npointes npointes.o $(LIBRARY)
+		$(FC) $(LFLAGS) -o npointes npointes.o $(LIBRARY)
 
 $(LIBRARY):	$(OBJS) $(AAOBJS)
 		ar cru $(LIBRARY) $(OBJS) $(AAOBJS)
-- 
2.47.1

